src/main/java/ch/njol/skript/expressions/ExprDamage.java

```diff
@@ -30,56 +31,76 @@
 import ch.njol.skript.doc.Events;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
-import ch.njol.skript.log.ErrorQuality;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Damage")
-@Description("How much damage is done in a damage event, possibly ignoring armour, criticals and/or enchantments. Can be changed (remember that in Skript '1' is one full heart, not half a heart).")
-@Examples({"increase the damage by 2"})
-@Since("1.3.5")
-@Events("damage")
+@Description({
+	"How much damage is done in a damage event, possibly ignoring armour, criticals and/or enchantments.",
```

Comment:
It may be worth clarifying that this supports item damage too. Does item damage ignore unbreaking? This description seems to indicate it does ignore whether unbreaking prevented it or not. I'm not sure, I'd just like to see a bit more clarity in this description.